docs/_writing-tests/rendering.md

```diff
@@ -9,6 +9,10 @@ these are especially using for [visual][] tests which need to be manually
 judged and following common patterns makes it easier to correctly tell if a
 given test passed or not.
 
+## Make Tests Self-Describing
```

Comment:
```
+ ## Make Tests Self-Describing
```

One final request: Change this so that only the first word of the heading is capitalized:

    ## Make tests self-describing

…because that’s the heading style used in the rest of this file (and I think throughout the rest of the docs.

(Sorry I should have caught that initially…)

Reply:
Sorry my bad i should also kept in mind.

Fixes now though :smile: